src/eslint-runner/reporter.js

```diff
@@ -43,7 +43,7 @@ class JestProgressBarReporter {
 
   onTestStart() /*: void */ {
     if (this._bar == null) {
-      this._bar = new ProgressBar(':bar :current/:total', {
+      this._bar = new ProgressBar(':bar :current/:total ', {
```

Comment:
Aha! I thought it's only my local problem. Could you please add there a simple comment explaining the motivation? Just to make sure we won't "fix it" back.

Reply:
✅